### PR TITLE
QA prompt for reader and UI changes

### PR DESCRIPTION
**Pull request by peteriz**
- Added QA prompt to the T5 reader
- Misc cosmetic demo UI fixes

### CODE REVIEW

File: fastrag/ui/webapp.py [hunk 1]
```diff
@@ -165,9 +178,25 @@ def reset_results(*args):
...
+    #     label_visibility="visible",
+    #     value=st.session_state.query_slctBox_val,
+    # )
+
+    # example_q = st.selectbox(
+    #     label="Examples",
+    #     options=tuple(EXAMPLES),
+    #     key="query_slctBox_val",
+    #     # on_change=set_example_as_query((st.session_state.query_slctBox_val))
+    # )
```
Comment (danielfleischer) [inline fastrag/ui/webapp.py:195]: Maybe delete this.

File: fastrag/readers/T5.py [hunk 2]
```diff
@@ -140,8 +163,8 @@ def __call__(self, tokenizer, query: str, documents: List[Document], top_k=None)
         preprocessed_doc = ". ".join([doc.content for doc in documents])
         # preprocessed_doc = [doc.content for doc in documents]
 
-        if self.mode == "Q&A":
-            prompt_with_content = f"question: {query} context: {preprocessed_doc}"
+        if self.mode == "qa":
```
Comment (danielfleischer) [inline fastrag/readers/T5.py:166]: Mode name needs to be consistent with https://github.com/IntelLabs/fastRAG/blob/c0eaa589e5d07fe019cf5152e3235eee6b26f578/config/reader/T5.yaml#L9  or define all the possible modes in a central place.

**Comment by peteriz** [comment]
@danielfleischer I addressed your comments. Please review.

**Review by danielfleischer — APPROVED** [review]
🛳️ it.

**Pull request merged**